src/main/java/com/mattermost/integration/figma/api/figma/file/service/FigmaFileServiceImpl.java

```diff
@@ -0,0 +1,48 @@
+package com.mattermost.integration.figma.api.figma.file.service;
+
+import com.mattermost.integration.figma.api.figma.file.dto.FigmaProjectFilesDTO;
+import com.mattermost.integration.figma.input.oauth.InputPayload;
+import com.mattermost.integration.figma.security.service.OAuthService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestTemplate;
+
+@Component
```

Comment:
Why are you not using service annotation?

Reply:
fixed